python/fbprophet/tests/test_diagnostics.py

```diff
@@ -44,7 +44,7 @@ def test_cross_validation(self):
 
         try:
             from dask.distributed import Client
-            client = Client()
+            client = Client(processes=False)  # noqa
```

Comment:
Is this an issue just with `setup.py test` or does it also impact if the user runs it with `processes=True` - assuming the timings you listed in  are for multiple processes with single threaded workers ?

Reply:
Yes, it's only failing with `setup.py test`. For context, this typically happens when have a callable that creates processes at the top-level of a file.

```
# file.py
pool = multiprocess.Pool()  # or dask.distributed.Client()
```

I don't really see why it would be happening in a test function.

And those timings were all for one-thread per worker.